src/context/navigationContext.js

```diff
@@ -0,0 +1,8 @@
+import { createContext } from 'react';
+
+const navigationContext = createContext({
+  navbarPages: [],
```

Comment:
Why do these need to be in a context? Aren't they static and can be imported as constants?

Reply:
We can't move these into easy to import functions without adding extra logic since we use the strings from our Redux store, thats why I'm defining them in a context so we can easily pass them down to any component below.

Otherwise we could export them from the constants file and add the strings key instead of the translated string.

Reply:
The old version just passed the navbarPages down to the Header component. Perhaps we can use the same approach and pass two props?

Reply:
I removed the context, see my latest commit.